src/app/clusters/laundry-washer-controls-server/laundry-washer-controls-server.cpp

```diff
@@ -192,3 +192,49 @@ void MatterLaundryWasherControlsPluginServerInitCallback()
     LaundryWasherControlsServer & laundryWasherControlsServer = LaundryWasherControlsServer::Instance();
     registerAttributeAccessOverride(&laundryWasherControlsServer);
 }
+
+Status MatterLaundryWasherControlsClusterServerPreAttributeChangedCallback(const chip::app::ConcreteAttributePath & attributePath,
+                                                                           EmberAfAttributeType attributeType, uint16_t size,
+                                                                           uint8_t * value)
+{
+    switch (attributePath.mAttributeId)
+    {
+    case Attributes::SpinSpeeds::Id:
+    case Attributes::SupportedRinses::Id:
+        return Status::UnsupportedWrite;
+    }
+    Delegate * delegate = GetDelegate(attributePath.mEndpointId);
+    if (delegate == nullptr)
+    {
+        return Status::Success;
+    }
+    switch (attributePath.mAttributeId)
+    {
+    case Attributes::SpinSpeedCurrent::Id: {
+        if (*value >= (uint8_t) delegate->GetGetSpinSpeedSize())
```

Comment:
We shouldn't have static_cast here.  If the size is expected to fit in uint8_t, and we have the extra API, then the API should return uint8_t.

Reply:
Will use GetSpinSpeedAtIndex() API